primo/opt_model/model_with_clustering.py

```diff
@@ -0,0 +1,421 @@
+#################################################################################
+# PRIMO - The P&A Project Optimizer was produced under the Methane Emissions
+# Reduction Program (MERP) and National Energy Technology Laboratory's (NETL)
+# National Emissions Reduction Initiative (NEMRI).
+#
+# NOTICE. This Software was developed under funding from the U.S. Government
+# and the U.S. Government consequently retains certain rights. As such, the
+# U.S. Government has been granted for itself and others acting on its behalf
+# a paid-up, nonexclusive, irrevocable, worldwide license in the Software to
+# reproduce, distribute copies to the public, prepare derivative works, and
+# perform publicly and display publicly, and to permit others to do so.
+#################################################################################
+
+# Standard libs
+import logging
+
+# Installed libs
+from pyomo.core.base.block import BlockData, declare_custom_block
```

Comment:
Tested this on the benchmark notebook. BlockData gives an import error. Replace with _BlockData

Reply:
@DevKakkad32 : I suspect this is a local import error for you. Could you please try reinstalling primo and pyomo? I don't see the ImportError you are seeing locally, and neither do I see it on GitHub Actions.